components/expression_language/caching.rst

```diff
@@ -39,12 +37,6 @@ in the object using the constructor::
     $cache = new MyDatabaseParserCache(...);
     $language = new ExpressionLanguage($cache);
```

Comment:
Can we maybe enhance this a bit, to show we're really doing PSR-6 here?

E.g. by showing something like:
```php
use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
use Symfony\Component\Cache\RedisAdapter;

$cache = new RedisAdapter(...);
$language = new ExpressionLanguage($cache);
```

Then, we can add a small "seealso" box pointing to the cache component:

```rst
.. seealso::

    See the :doc:`/components/cache` documentation for more information about
    available cache adapters.
```